MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -469,6 +470,20 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                 }
                 break;
             }
+            case MSIDFamilyRefreshTokenType:
+            {
+                MSIDRefreshToken *refreshToken = (MSIDRefreshToken *) token;
```

Comment:
should we do "isKindOfClass" before casting?

Reply:
The switch is reading `credentialType` from each individual class; but it was casting to the base class instead of the right class. Updated